sdks/python/apache_beam/testing/datatype_inference_test.py

```diff
@@ -174,6 +175,9 @@ def test_infer_typehints_schema(self, _, data, schema):
   @parameterized.expand([(d["name"], d["data"], d["pyarrow_schema"])
                          for d in TEST_DATA])
   @unittest.skipIf(pa is None, "PyArrow is not installed")
+  @unittest.skipIf(
+      sys.platform == "win32",
+      "numpy array dtype is coming as int32 by default in a windows 64 bit")
```

Comment:
How does this test fail? We need a JIRA so that we understand the implications of this for the users? Does this fail on all Python versions?